### ISSUE TITLE
Fix parsing error for `list:hosts`

### ISSUE THREAD

**Issue by stmh** [factorial-io/phabalicious#145]
```
PHP Parse error:  syntax error, unexpected ')' in phar:///usr/local/bin/phab/src/Method/ResticMethod.php on line 185

```
when using older php versions.

**Comment by alvar0hurtad0** [comment]
Hello @stmh this error is actually in more commands, could you release the fix?

**Comment by stmh** [comment]
Fixed with 3.6.1
